### PR TITLE
Added products in Cape Town, updated hierarchical layer set up etc

### PR DESCRIPTION
**Pull request by fangfy**
* Updated OWS catalog with new products in Cape Town
* Updated how hierarchical layers are set up, so we can have flexibility to add daily and annual products
* Also updated DEAfrica contact information

### CODE REVIEW

File: services/ows_refactored/dev_af_ows_root_cfg.py [hunk 1]
```diff
@@ -73,7 +73,7 @@
     },  # END OF global SECTION
     "wms": {
         # Config for WMS service, for all products/layers
-        "s3_url": "https://data.digitalearth.africa",
+        # "s3_url": "https://data.digitalearth.africa",
```
Comment (fangfy) [inline services/ows_refactored/dev_af_ows_root_cfg.py:76]: can i remove this?

File: services/ows_refactored/dev_af_ows_root_cfg.py [hunk 2]
```diff
@@ -73,7 +73,7 @@
...
         "s3_bucket": "deafrica-data",
```
Comment (fangfy) [inline services/ows_refactored/dev_af_ows_root_cfg.py:77]: we have multiple buckets now, shall we remove this?

**Review by fangfy — COMMENTED** [review]
Blocked until S1 layer is added.

**Comment by pindge** [comment]
The reason the test is failing is because you are introducing more product layers, u need to update `inventory.json`  for this test to pass.

**Comment by fangfy** [comment]
> The reason the test is failing is because you are introducing more product layers, u need to update `inventory.json` for this test to pass. https://github.com/digitalearthafrica/config/tree/master/services/inventory
> 
> this json file keeps accounts of all the layers that are expected, in certain cases some change could make ows config to lose layers either due to error or wrong deletion, keeping this updated is for peace of mind.

inventory.json has been updated. Still getting errors.

File: services/ows_refactored/dev_af_ows_root_cfg.py (hunk 1 above)
Comment (SpacemanPaul) [reply services/ows_refactored/dev_af_ows_root_cfg.py:76]: The s3_url and s3_bucket entries are only used for converting the links to the underlying data files into pretty http links for the GetFeatureInfo responses.

Removing will leave you with broken URLs - but that looks like something I'm going to have handle within OWS.

**Review by SpacemanPaul — APPROVED** [review]
Looks good to me.

There are some issues with the S3 urls, but I'll need to fix them OWS rather than here.

**Comment by pindge** [comment]
> > The reason the test is failing is because you are introducing more product layers, u need to update `inventory.json` for this test to pass. https://github.com/digitalearthafrica/config/tree/master/services/inventory
> > this json file keeps accounts of all the layers that are expected, in certain cases some change could make ows config to lose layers either due to error or wrong deletion, keeping this updated is for peace of mind.
> 
> inventory.json has been updated. Still getting errors.

you can click on `details` and it will take u to the error, I have extracted it for you here 
```
{'values_changed': {"root['total_layers_count']": {'new_value': 13, 'old_value': 14}}, 'iterable_item_removed': {"root['layers'][2]": {'product': ['ga_ls8c_wofs_2'], 'styles_count': 2, 'styles_list': ['observations', 'wet']}}}
```
basically, you have `ga_ls8c_wofs_2` is either not loading correctly or its not there

**Pull request merged**